### DISCUSSION TITLE
Request drop down for tests should include what the verb is

### DISCUSSION THREAD

**Discussion by davehouser1** [Kong/insomnia#6515] [Ideas]
When going to tests, the requests drop down only shows the API names. I use the same name for my API across different verbs (GET, POST, PUT, DELETE, etc.)
The problem is the dropdown just has the same name for all my requests, there is no for me to tell which one is which, without adding "GET" to the actual name of the request name. I find this to be redundant as the name shows up with the verb to the right of it in the DEBUG page. 

EDIT: TBH you should just replicate the same directory tree structure with verbs included that you see in Debug. This fixes a few problems.
1) As I said before, get to see which verbs each request uses.
2) Having directory structure saves me the time of scrolling through every request I ever made I can just go to specific folder / subfolders
3) I may have the same verb + name in another directory, which, again shows up twice in the list.

**Comment by kobenguyent** [comment]
I have submitted this PR https://github.com/Kong/insomnia/pull/6530 for that purpose
